src/sentry/monitors/endpoints/organization_monitor_details.py

```diff
@@ -87,6 +96,11 @@ def put(self, request: Request, organization, project, monitor) -> Response:
         """
         Update a monitor.
         """
+        # set existing values as validator will overwrite
+        existing_config = monitor.config
+        existing_margin = existing_config.get("checkin_margin")
+        existing_max_runtime = existing_config.get("max_runtime")
```

Comment:
it looks like the `validator` already saves the values...we might not need to update most of the params in the query below

Reply:
Wait what do you mean?